HW4_Bobkov/protein_tools.py

```diff
@@ -0,0 +1,220 @@
+def compare(sequences: list, round_dec=3, percentages=False)->dict:
+    """
+    Compare aminoacids between reference sequence and other sequences
+    arguments:
+    - sequences (list): reference sequence and other sequences for comparison
+    - round_dec (int): a number of decimals to round the number to
+    - percentages (bool): whether percentages are returned instead of fractions
+    return:
+    - comparisons (dict): dictionary with compared sequences as keys and percentages/fractions as their values
+    """
+    comparisons={}
+    for seq in range(1,len(sequences)):
```

Comment:
Так,
1) У вас итерируемая переменная не сиквенс же, а индекс. Как переменная seq может хранить индекс?
2) Но тут итерироваться можно по сиквенсам, а не по индексам:
```suggestion
    for seq in sequences:
```